packages/next/src/server/stream-utils/node-web-streams-helper.ts

```diff
@@ -58,6 +43,17 @@ export function streamFromString(str: string): ReadableStream<Uint8Array> {
   })
 }
 
+export function concatStreams<T>(
+  first: ReadableStream<T>,
+  second: ReadableStream<T>
+) {
+  const { readable, writable } = new TransformStream<T, T>()
+  first.pipeTo(writable, { preventClose: true }).then(() => {
+    second.pipeTo(writable)
+  })
```

Comment:
`.pipeTo` returns a `Promise`, which should have a `.catch` attached to it to avoid a `UnhandledPromiseRejectionWarning`.

Is there a reason why the above `chainStreams` doesn't do what's required? The upside to `chainStreams` is that it handles the case where the `pipeTo` rejects.